web/templates/email/weekly_info.html.eex

```diff
@@ -0,0 +1,20 @@
+<h1>Thanks <%= @attendee.name %>!</h1>
+
+<p>Here is your connection card for the week of <%= @weekly_info.week_date %>:</p>
+<ul>
```

Comment:
Design thing:

Do you think this information here is useful to the person you're emailing? I know what my prayer request was so it seems odd to just return that to me.